src/admin/class-wordlift-install-wizard.php

```diff
@@ -0,0 +1,759 @@
+					margin:0 auto;
+					display:block;
+				}
+				
+				#deletelogo {
+					position:absolute;
+					top:-10px;
+					right:-10px;
+					background:white;
+					color:#2e92ff;
+					font-size: 20px;
+					border-radius: 20px;
+					border: 1px #2e92ff solid;
+					display: block;
+					padding: 0 2px;
+				}
+				
+			</style>
+			<?php 
+				wp_enqueue_script('jquery');
+				wp_enqueue_media(); 
+				wp_enqueue_style('common');
+				wp_print_styles();
+				wp_print_scripts();
+			?>
+		</head>
+		<body>
+			<div class="wl-setup">
+				<a id="close" title="<?php _e('Exit the wizard','wordlift')?>" href="<?php echo admin_url()?>"><span class="fa fa-times"></span></a>
+				<div id="wl-title">
+					<div id="wl-logo">
+						<span class="bold">Word</span>Lift
+					</div>
+					<div id="bullets">
+						<span class="bullet" data-step="welcome"></span>
+						<span class="bullet" data-step="license"></span>
+						<span class="bullet" data-step="vocabulary"></span>
+						<span class="bullet" data-step="language"></span>
+						<span class="bullet" data-step="publisher"></span>
+					</div>
+					<div id="topright">
+					</div>
+					<div style="clear:both"></div>
+				</div>
+		<?php
+	}
+
+	/**
+	 * Output the html for the footer of the page
+	 *
+	 * @since    3.9.0
+	 *
+	 */
+	public function footer( ) {
+		?>
+			</div>
+			<?php do_action( 'admin_footer');?>
+			</body>
+		</html>
+		<?php
+	}
+	
+	/**
+	 * Output the html for the welcome page
+	 *
+	 * @since    3.9.0
+	 *
+	 */
+	public function welcome_page() {
+		?>
+		<div id="title"><?php _e('Welcome','wordlift')?></div>
+		<div id="message"><?php _e('Thank you for downloading WordLift. Now you can boost your website with double digit growth.','wordlift')?></div>
+		<div id="buzzcont">
+			<div class="buzz"><span class="fa fa-university"></span><?php _e('Trustworthiness','wordlift')?></div>
+			<div class="buzz"><span class="fa fa-map-marker"></span><?php _e('Enrichment','wordlift')?></div>
+			<div class="buzz"><span class="fa fa-heart"></span><?php _e('Engagement','wordlift')?></div>
+			<div class="buzz"><span class="fa fa-hand-o-right"></span><?php _e('Smart Navigation','wordlift')?></div>
+			<div class="buzz"><span class="fa fa-google"></span><?php _e('SEO Optimization','wordlift')?></div>
+			<div class="buzz"><span class="fa fa-group"></span><?php _e('Content Marketing','wordlift')?></div>
+			<div style="clear:both">
+		</div>
+		<div id="buttons">
+			<a href="https://wordlift.io/blogger" target="_tab" class="button-primary"><?php _e( 'Learn More', 'wordlift' ); ?></a>
+			<a id="nextstep" href="<?php echo esc_url( admin_url( 'admin.php?page=wl-setup&step=license' ) ); ?>"><?php _e( 'Get started', 'wordlift' ); ?></a>
+		</div>
+		<?php
+	}
+	
+	/**
+	 * Output the html for the license page
+	 *
+	 * @since    3.9.0
+	 *
+	 */
+	public function license_page() {
+		$key = '';
+		if (isset($_COOKIE['wl_key']))
+			$key = $_COOKIE['wl_key'];
+		$valid = 'invalid';
```

Comment:
Can be simplified with `$valid = $this->key_validation_service->is_valid( $key ) ? 'valid' : 'invalid';` (using the new `Wordlift_Key_Validation_Service`).